aten/src/ATen/Context.h

```diff
@@ -385,8 +385,11 @@ class TORCH_API Context {
       ? at::LinalgBackend::Cusolver
       : at::LinalgBackend::Default;
   at::BlasBackend blas_preferred_backend =
-      (c10::utils::check_env("TORCH_BLAS_PREFER_CUBLASLT") == true ||
-       c10::utils::check_env("TORCH_BLAS_PREFER_HIPBLASLT") == true)
+#ifdef USE_ROCM
+      (c10::utils::check_env("TORCH_BLAS_PREFER_HIPBLASLT") != false)
```

Comment:
Might be cleaner to have both checks the same, either == true or != false.

Reply:
yeah it's intentional. The behavior of check_env is that it'll return nullopt when the env var is not set (the most common case), and true if 1, false if 0: https://github.com/pytorch/pytorch/blob/ca561d639b40a9fce088262e2fb35c7dfb61d588/c10/util/env.h#L16. Therefore checking != false makes true as the default, and == true will make false as the default.